python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -874,7 +874,6 @@ def _fast_slow_function_call(
     func: Callable,
     /,
     *args,
-    **kwargs,
 ) -> Any:
     """
     Call `func` with all `args` and `kwargs` converted to their
```

Comment:
Could you update this docstring to note that `args` contains `func`'s args and kwargs?